test/test_other.py

```diff
@@ -3204,7 +3204,7 @@ def test_embind_return_value_policy(self):
 
   def test_jspi_wildcard(self):
     self.require_jspi()
-    self.emcc_args += ['-sASYNCIFY_EXPORTS=async*']
+    self.emcc_args += ['-sJSPI_EXPORTS=async*']
```

Comment:
Is this the only place we use this in the whole test suite?

Reply:
There are three others. I've left them though so we still test that option until we remove it. I suppose I could just leave one, if that's preferred.

Reply:
Yes, for deprecated settings I'd prefer that all tests use the new thing.. and we have a single test for the deprecated one (can be version of an existing test).